bodhi/client/__init__.py

```diff
@@ -25,6 +25,7 @@
 import traceback
 import re
 import functools
+import munch
```

Comment:
So this is a third party library, and PEP-8 says third party libraries should be grouped together, so let's move this to be with click and six below.

Reply:
@vismay-golwala let's update this then we can merge this PR :monkey_face: